data/scenarios/02Tutorials/04-build.yaml

```diff
@@ -0,0 +1,64 @@
+name: Build
+description: |
+  Learn how to build robots to carry out simple tasks.
+#
+# Has to be easy, so that you do not have to debug it too much yet.
+#
+entities:
+  - name: Goal
+    display:
+      attr: device
+      char: 'X'
+    description:
+    - Now that you know basic commands, it is time to
+      build robots to do the work for you.
+    - Let's start by building a one task gardener robot.
+    - You can build a robot with 'build {COMMANDS}',
+      where instead of 'COMMANDS' you write the sequence
+      of commands for robot to execute (separated by ';').
+    - Build a robot to grab the flower and place it next
+      to the water.
+
+win: |
+  def flowerAt = \p. teleport self p; ishere "flower"; end;
```

Comment:
Would it be worth adding a god-mode command to find out what is at an arbitrary location?

Reply:
Well as long as it's not a blocking entity like a mountain, you can just teleport there and check.
This helper function has a fairly low overhead. A grab-from-arbitrary-place could be useful though.

Reply:
I am closing this as unnecessary right now, if we need it later we can easily add it.